.github/workflows/release-build-publish.yml

```diff
@@ -100,6 +100,8 @@ jobs:
           platforms: linux/amd64,linux/arm64
           tags: |
             amazon/aws-xray-daemon:${{ github.event.inputs.version }}
+            amazon/aws-xray-daemon:latest
+            amazon/aws-xray-daemon:3.x
```

Comment:
We would have to change this value for major version releases in the future otherwise we would push breaking changes to 3.x customers.
Okay for now but will need to keep this in mind.

Reply:
Yeah good call out.